src/components/templates/Camp/hooks/usePartygoersOverlay.tsx

```diff
@@ -13,14 +13,11 @@ interface UsePartygoersOverlay {
   withMiniAvatars?: boolean;
   rows: number;
   columns: number;
-  partygoers?: readonly WithId<User>[];
+  partygoers: readonly WithId<User>[];
   setSelectedUserProfile: (user: WithId<User>) => void;
 }
 
-export type UsePartygoersReturn =
-  | ("" | JSX.Element)[]
-  | JSX.Element
-  | undefined;
+export type UsePartygoersReturn = (null | JSX.Element)[] | null;
```

Comment:
Since React allows `null` to be returned in JSX, is there a JSX type that already includes it, such that we don't need to explicitly add it here? One that says "anything that is valid in a React JSX-type position"